<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgSettingsRabbitVNC</class>
 <widget class="QDialog" name="CDlgSettingsRabbitVNC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting RabbitVNC</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/image/App</normaloff>:/image/App</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>Ok(&amp;O)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel(&amp;C)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Server">
      <attribute name="title">
       <string>Server</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leName">
           <property name="toolTip">
            <string>Connecter name</string>
           </property>
           <property name="statusTip">
            <string>Connecter name</string>
           </property>
           <property name="whatsThis">
            <string>Connecter name</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lbServer">
           <property name="text">
            <string>Server:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leServer">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Port:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spPort">
           <property name="maximum">
            <number>65536</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>User name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leUserName"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lePassword">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbShow">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="eye-on"/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QCheckBox" name="cbShared">
           <property name="text">
            <string>Shared</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbRealTimeUpdate">
           <property name="text">
            <string>Real time update</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbOnlyView">
           <property name="text">
            <string>Only view</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QCheckBox" name="cbLocalCursor">
           <property name="text">
            <string>Show local cursor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbClipboard">
           <property name="text">
            <string>Enable clipboard</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="cbSave">
           <property name="text">
            <string>Save password</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="cbResizeWindows">
           <property name="text">
            <string>Support windows resize</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="cbShowServerName">
         <property name="text">
          <string>Show server name</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QGroupBox" name="gpIce">
         <property name="title">
          <string>Ice:</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Peer user:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lePeerUser"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Proxy">
      <attribute name="title">
       <string>Proxy</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="3">
          <widget class="QSpinBox" name="spProxyPort">
           <property name="maximum">
            <number>65537</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Port:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Server:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="leProxyServer"/>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="rbProxyNo">
           <property name="text">
            <string>No</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbProxySocks">
           <property name="text">
            <string>Socks5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbHttp">
           <property name="text">
            <string>Http</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>User name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leProxyUser"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leProxyPassword"/>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Compress">
      <attribute name="title">
       <string>Compress</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QGroupBox" name="gpColorLevel">
         <property name="title">
          <string>Color level</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="1" column="0">
           <widget class="QRadioButton" name="rbFull">
            <property name="text">
             <string>Full</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rbLow">
            <property name="text">
             <string>Low</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rbMeduim">
            <property name="text">
             <string>Meduim</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QRadioButton" name="rbVeryLow">
            <property name="text">
             <string>Very low</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QSpinBox" name="spCompressLevel">
           <property name="maximum">
            <number>9</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>0=fast, 9=best</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QCheckBox" name="cbJPEG">
           <property name="text">
            <string>Enable JPEG compress</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spJPEGLevel">
           <property name="maximum">
            <number>9</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>0=Worst, 9=best</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" rowspan="3">
        <widget class="QGroupBox" name="gpEncodeing">
         <property name="title">
          <string>preferred Encoding</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rbTight">
            <property name="text">
             <string>Tight</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rbHextile">
            <property name="text">
             <string>Hextile</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QRadioButton" name="rbRaw">
            <property name="text">
             <string>Raw</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rbZRLE">
            <property name="text">
             <string>ZRLE</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rbCoRRE">
            <property name="text">
             <string>CoRRE</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QRadioButton" name="rbRRE">
            <property name="text">
             <string>RRE</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QRadioButton" name="rbCopyRect">
            <property name="text">
             <string>copyRect</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="cbCompress">
         <property name="text">
          <string>Compress level:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="cbCompressAutoSelect">
         <property name="text">
          <string>Auto select</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leServer</tabstop>
  <tabstop>leUserName</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>cbShared</tabstop>
  <tabstop>cbRealTimeUpdate</tabstop>
  <tabstop>cbLocalCursor</tabstop>
  <tabstop>cbClipboard</tabstop>
  <tabstop>cbSave</tabstop>
  <tabstop>cbResizeWindows</tabstop>
  <tabstop>cbOnlyView</tabstop>
  <tabstop>pbOk</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>cbCompressAutoSelect</tabstop>
  <tabstop>rbTight</tabstop>
  <tabstop>rbZRLE</tabstop>
  <tabstop>rbHextile</tabstop>
  <tabstop>rbCoRRE</tabstop>
  <tabstop>rbRRE</tabstop>
  <tabstop>rbCopyRect</tabstop>
  <tabstop>rbRaw</tabstop>
  <tabstop>rbFull</tabstop>
  <tabstop>rbMeduim</tabstop>
  <tabstop>rbLow</tabstop>
  <tabstop>rbVeryLow</tabstop>
  <tabstop>cbCompress</tabstop>
  <tabstop>spCompressLevel</tabstop>
  <tabstop>cbJPEG</tabstop>
  <tabstop>spJPEGLevel</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
